### PR TITLE
Add overloaded constructors for optional apn, login and pass in NBConnectionHandler

### PR DESCRIPTION
**Pull request by AlbyIanna**

### CODE REVIEW

File: src/Arduino_NBConnectionHandler.cpp [hunk 1]
```diff
@@ -113,7 +123,7 @@ void NBConnectionHandler::update() {
       case NetworkConnectionState::CONNECTING: {
           // NOTE: Blocking Call when 4th parameter == true
           NB_NetworkStatus_t networkStatus;
-          networkStatus = gprs.attachGPRS();
+          networkStatus = gprs.attachGPRS(apn, login, pass, true);
```
Comment (sandeepmistry) [inline src/Arduino_NBConnectionHandler.cpp]: I don't see a default value for `apn`, `login`, or `pass` do they default to NULL or empty string?

Comment (AlbyIanna) [reply src/Arduino_NBConnectionHandler.cpp]: not sure, but I think they're `NULL`

Comment (sandeepmistry) [reply src/Arduino_NBConnectionHandler.cpp]: I highly recommend they be explicitly initialized.

Comment (AlbyIanna) [reply src/Arduino_NBConnectionHandler.cpp]: How would you initialize those values? Can I do that in the signature?

Comment (sandeepmistry) [reply src/Arduino_NBConnectionHandler.cpp]: Also I don’t think this compiles, here is the signature: https://github.com/arduino-libraries/MKRNB/blob/master/src/GPRS.h#L58

The parameters should be passed to ‘nbAccess.begin(...)’.

**Pull request merged**